examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/transformers/weight_only/text-generation/requirements_cpu_woq.txt

```diff
@@ -15,3 +15,4 @@ lm-eval==0.4.3
 huggingface_hub
 numba
 tbb
+intel-extension-for-pytorch
```

Comment:
do we have version dependency or min version dependency?

Reply:
> do we have version dependency or min version dependency?

Yes, it depends on the torch version, we need to update the torch version at the same time.
BTW is it necessary to freeze the torch version?

Reply:
I think it is ok to keep current verified torch version unless we have reason to move.  for IPEX even without specifying the version.  IPEX 2.4 version will be used to match torch, am I right?

Reply:
> I think it is ok to keep current verified torch version unless we have reason to move. for IPEX even without specifying the version. IPEX 2.4 version will be used to match torch, am I right?

NO, it will install the latest IPEX version

Reply:
if that's the case, I would suggest keep IPEX and torch version matched.